invisible_cities/io/dst_io.py

```diff
@@ -1,10 +1,12 @@
 import tables as tb
 import pandas as pd
+import numpy  as np
 from tables import NoSuchNodeError
 from tables import HDF5ExtError
 import warnings
 
-
+from .. reco import tbl_functions as tbl
+from .  table_io           import make_table
 
```

Comment:
Review alignment and spaces in the whole PR.

Reply:
I tried but I am not sure I did it right, I think I still dont get the alignment style we use

Reply:
Use alignment to highlight symmetries and patterns. Anything that makes it easier for the reader to spot things that are similar or belong together, and anything that helps differences to stand out.

In the case of imports, the most obvious thing that always needs to be done is to align the import keywords: `from`, `import` and `as`. Align the names which are being bound, which may require a nanosecond's thought in the absence/precesce of `as`. For example:

    from vegetable import aubergine as eggplant
    from fruit     import banana    as platano
    from fruit     import              orange

(If you are reading the above in your mail client, who knows what alignment you are seeing: Make sure you read it in a fixed-width font!)

Aligning `orange` with `aubergine` and `banana` would be OK, as it belongs in both the column of components being imported and in the column of names being bound. (Strictly speaking this means that we should write `from fruit import orange as orange`, but I think that would be taking it too far.)

In the case of names which share a common pattern at the end or in the middle, align the common parts:

    from a import            a_thingy
    from a import         bbbb_thingy
    from a import cccccccccccc_thingy

in order to highlight that we are importing 3 thingies.

Reply:
Thanks for the reply! What happens when I have unrelated imports ? As:
`
import warnings
from .. core.exceptions    import TableMismatch
from .  table_io                import make_table
from typing                      import Optional
`
what should I try to match in this example?

Reply:
The first line goes in a separate block from the rest. The last 3 have the same structure consisting of 5 columns:

1. `from`
2. relative path
3. module
4. `import`
5. component

Align each column. (The last line has an empty 2nd column.)